doc/modules/manifold.rst

```diff
@@ -387,10 +387,10 @@ The overall complexity of standard LTSA is
 
 .. topic:: References:
 
-   * `"Principal manifolds and nonlinear dimensionality reduction via
+   * :doi:`"Principal manifolds and nonlinear dimensionality reduction via
      tangent space alignment"
```

Comment:
The link gives a not found... apparently this DOI is an internal DOI for citeseerx?
I can find the paper at the standard DOI with this identification: 10.1137/S1064827502419154

Reply:
I'm sorry for the inconveniences I have caused with my last exhaustive commit. I am about to fix all links, where possible replace citeseerx with either doi or arxiv, and separate the rst files to a new PR. Will let you know, when I'm done.